src/target/riscv/riscv-013.c

```diff
@@ -2845,15 +2845,35 @@ static int execute_fence(struct target *target)
 
 	/* FIXME: For non-coherent systems we need to flush the caches right
 	 * here, but there's no ISA-defined way of doing that. */
+	int result;
```

Comment:
it looks like we don't really need this `result` variable here. @wxjstz maybe we could cleanup this?